recipes/fastplotlib/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.3.0" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: fastplotlib
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/f/fastplotlib/fastplotlib-{{ version }}.tar.gz
+  sha256: d5b6c6e43736178b0d68fe02abc7df93ccc5077afcdccbba86558de739e27cfe
```

Comment:
```suggestion
  sha256: 42876e94fd8d9f2499d32e723cc975c6672a76f2876d9f3c4118cfa9998cd3f2
````

Reply:
forgot to add the new hash https://pypi.org/project/fastplotlib/0.3.0/#fastplotlib-0.3.0.tar.gz